### ISSUE TITLE
竖向数字识别效果很差

### ISSUE THREAD

**Issue by Mason1992-Git** [PaddlePaddle/PaddleOCR#8799]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
问题描述:竖向数字识别效果很差
系统环境：win10 paddleocr2.6
需求识别横向及竖向的数字和英文字母，待识别的图像像素大小大概为30~120，长宽比为1:1.5左右：
![S_1_11](https://user-images.githubusercontent.com/79744902/211012481-d8595046-0eab-43d2-818e-b880a9a26123.jpg)
![H_1_02](https://user-images.githubusercontent.com/79744902/211012576-831818de-c1fc-408a-af34-099be5ca09c5.jpg)
目前采用方法:使用yolov5进行文字区域检测后，判断图像长宽比，如为竖向的，则逆时针旋转90后输入识别模型，横向则直接输入识别模型。使用en_PP-OCRv3_rec_train作为预训练模型进行微调，en_PP-OCRv3_rec.yml为配置文件(仅修改了相关路径)，en_dict.txt为字典。
训练集制作:将竖向逆时针旋转90后与横向一起加入训练。数据集一共1879张，其中横竖比例大致为1:1如图：
![image](https://user-images.githubusercontent.com/79744902/211013494-6cc1b849-478b-4a4a-8826-9fcc98a5d86d.png)
yml文件:
Global:
  debug: false
  use_gpu: true
  epoch_num: 500
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: F:\Paddle_Ocr\PaddleOCR-release-2.6\output/v3_en_mobile
  save_epoch_step: 3
  eval_batch_step:
  - 0
  - 2000
  cal_metric_during_train: true
  pretrained_model: F:\Paddle_Ocr\PaddleOCR-release-2.6/pretrain_models/en_PP-OCRv3_rec_train/best_accuracy.pdparams
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: F:\Paddle_Ocr\PaddleOCR-release-2.6\ppocr\utils/en_dict.txt
  max_text_length: 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3_en.txt
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0001
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 3.0e-05
Architecture:
  model_type: rec
  algorithm: SVTR
  Transform: null
  Backbone:
    name: MobileNetV1Enhance
    scale: 0.5
    last_conv_stride:
    - 1
    - 2
    last_pool_type: avg
  Head:
    name: MultiHead
    head_list:
    - CTCHead:
        Neck:
          name: svtr
          dims: 64
          depth: 2
          hidden_dims: 120
          use_guide: true
        Head:
          fc_decay: 1.0e-05
    - SARHead:
        enc_dim: 512
        max_text_length: 25
Loss:
  name: MultiLoss
  loss_config_list:
  - CTCLoss: null
  - SARLoss: null
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: false
Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/
    ext_op_transform_idx: 1
    label_file_list:
    - F:\Paddle_Ocr\PaddleOCR-release-2.6\train_data\rec/rec_gt_train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape:
        - 48
        - 320
        - 3
        max_text_length: 25
    - RecAug: null
    - MultiLabelEncode: null
    - RecResizeImg:
        image_shape:
        - 3
        - 48
        - 320
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 128
    drop_last: true
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data
    label_file_list:
    - F:\Paddle_Ocr\PaddleOCR-release-2.6\train_data\rec/rec_gt_train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode: null
    - RecResizeImg:
        image_shape:
        - 3
        - 48
        - 320
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 128
    num_workers: 4
profiler_options: null

**Comment by Mason1992-Git** [comment]
是我测试问题，打扰了
